### PR TITLE
Add test case for feature-1

### PR DESCRIPTION
**Pull request by Zerl1990**

### CODE REVIEW

File: TEST-FEATURE-1.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Step: Login to system
+step: Got to settings
```
Comment (Zerl1990) [inline TEST-FEATURE-1.md]: Missing corner corner cases to validate
